src/sentry/static/sentry/app/views/health/list/healthTableRow.tsx

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable react/prop-types */
+// TODO: we should probably disable eslint rule react/prop-types for typescript functional components
```

Comment:
Instead of disabling this lint rule, we can just explicitly type the FC parameters with `Props`